tests/scores/test_weights.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import scores.continuous
+
+ZERO = np.array([[1 for i in range(10)] for j in range(10)])
+
+
+# Standard forecast and observed test data which is static and can be used
+# across tests
+np.random.seed(0)
+lats = [50, 51, 52, 53]
+lons = [30, 31, 32, 33]
+fcst_temperatures_2d = 15 + 8 * np.random.randn(1, 4, 4)
+obs_temperatures_2d = 15 + 6 * np.random.randn(1, 4, 4)
+FCST_2D = xr.DataArray(fcst_temperatures_2d[0], dims=["latitude", "longitude"], coords=[lats, lons])
+OBS_2D = xr.DataArray(obs_temperatures_2d[0], dims=["latitude", "longitude"], coords=[lats, lons])
+IDENTITY = np.ones((4, 4))
+ZEROS = np.zeros((4, 4))
+
+
+def simple_da(data):
+    """
+    Helper function for making a DataArray with a latitude coordinate variable roughly
+    over the Australian region
+    """
+    lats = np.arange(-5, -45, (-45 / len(data)))
+    arr = xr.DataArray.from_dict(
+        {
+            "coords": {
+                "lat": {"data": lats, "dims": "lat"},
+            },
+            "data": data,
+        }
+    )
+    return arr
+
+
+# These scores will be tested for valid processing of weights
+all_scores = [scores.continuous.mse, scores.continuous.mae]
+
+
+def test_weights_identity():
+    for score in all_scores:
+        unweighted = score(FCST_2D, OBS_2D)
+        weighted = score(FCST_2D, OBS_2D, weights=IDENTITY)
+        assert unweighted == weighted
+
+
+def test_weights_zeros():
+    for score in all_scores:
+        unweighted = score(FCST_2D, OBS_2D)
+        weighted = score(FCST_2D, OBS_2D, weights=ZEROS)
+
+        assert unweighted != weighted
+        assert weighted.sum() == 0
+
+
+def test_weights_latitude():
+    """
+    Tests the use of latitude weightings, not the correctness
```

Comment:
Update this comment now that you do test for correctness below.

Reply:
Done, thanks